packages/flutter_webrtc/tizen/src/flutter_webrtc_tizen_plugin.cc

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
```

Comment:
There is a LICENSE file that is compatible with the MIT license in the root directory. I am not sure, but  you should add a LICENSE file for BSD.

Reply:
Thanks, that's a good point. I added LICENSE file like you said.

Reply:
Hmm... that's strange.. I don't know if it's a problem like that, but I'll check it out

Reply:
Thanks for the quick response.